packages/adapters-library/src/core/provider/CustomJsonRpcProvider.ts

```diff
@@ -81,9 +80,17 @@ export class CustomJsonRpcProvider extends JsonRpcProvider {
   async getStableBlockNumber(): Promise<number> {
     const currentBlockNumber = await this.getBlockNumber()
 
-    const blockNumberTenMinsAgo =
-      currentBlockNumber - (AVERAGE_BLOCKS_PER_10_MINUTES[this.chainId] ?? 0) // default to 0 to avoid failures
-    return blockNumberTenMinsAgo
+    const blocksToMoveBack = (() => {
+      switch (this.chainId) {
+        case Chain.Arbitrum:
+          return AVERAGE_BLOCKS_PER_10_MINUTES[this.chainId] / 4
+        default:
+          return AVERAGE_BLOCKS_PER_10_MINUTES[this.chainId]
+      }
+    })()
```

Comment:
Changed stable block for arbitrum to 2.5m.

Not massively better, but spamming postman gave me the impression that the closer the block, the less chance of it taking an unreasonable amount of time.